src/toil/test/provisioners/cgcloud/provisionerTest.py

```diff
@@ -265,3 +266,44 @@ def testAutoScaledCluster(self):
     @integrative
     def testAutoScaledSpotCluster(self):
         self._test(autoScaled=True, spotInstances=True)
+
+
+restartScript = """\"from toil.job import Job
```

Comment:
Can you use this pattern instead. It's a bit unconventional but I like it because it applies syntax checking to embedded scripts:

https://github.com/BD2KGenomics/toil/blob/master/src/toil/test/src/hotDeploymentTest.py#L44
https://github.com/BD2KGenomics/toil/blob/master/src/toil/test/__init__.py#L718